src/__tests__/server.test.ts

```diff
@@ -74,3 +76,40 @@ test('stop twice does nothing on second attempt', async () => {
 
   fail('expected a connection error as server should be closed');
 });
+
+const getPort = async (options?: Options) => {
```

Comment:
It seems this method is unnecessary, as all it's doing is using `availablePort()` to find a vacant port. You can achieve the same effect by using the value 0 as a port, just like the 'server starts with random port' test is doing.